web/angular-wallet/src/app/main/assets/assets.component.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {AssetsComponent} from './assets.component';
+import { MatButtonModule } from '@angular/material/button';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatSortModule } from '@angular/material/sort';
+import { MatTableModule } from '@angular/material/table';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import {ReactiveFormsModule} from '@angular/forms';
+import {I18nModule} from 'app/layout/components/i18n/i18n.module';
+import {BehaviorSubject} from 'rxjs';
+import {StoreService} from 'app/store/store.service';
+import {I18nService} from 'app/layout/components/i18n/i18n.service';
+import {AccountService} from 'app/setup/account/account.service';
+import {RouterTestingModule} from '@angular/router/testing';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {PageModule} from '../../components/page/page.module';
```

Comment:
some people from twitter are apparently triggered by our lack of a prettier rule for this.

Reply:
?

Reply:
> some people from twitter are apparently triggered by our lack of a prettier rule for this.

? Don't understand what you mean